staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler_test.go

```diff
@@ -418,3 +439,246 @@ func TestRouting(t *testing.T) {
 		})
 	}
 }
+
+func TestHandlerConversionWithWatchCache(t *testing.T) {
+	testHandlerConversion(t, true)
+}
+
+func TestHandlerConversionWithoutWatchCache(t *testing.T) {
+	testHandlerConversion(t, false)
+}
+
+func testHandlerConversion(t *testing.T, enableWatchCache bool) {
+	cl := fake.NewSimpleClientset()
+	informers := informers.NewSharedInformerFactory(fake.NewSimpleClientset(), 0)
+	crdInformer := informers.Apiextensions().V1().CustomResourceDefinitions()
+
+	server, storageConfig := etcd3testing.NewUnsecuredEtcd3TestClientServer(t)
+	defer server.Terminate(t)
+
+	crd := multiVersionFixture.DeepCopy()
+	if _, err := cl.ApiextensionsV1().CustomResourceDefinitions().Create(context.TODO(), crd, metav1.CreateOptions{}); err != nil {
+		t.Fatal(err)
+	}
+	if err := crdInformer.Informer().GetStore().Add(crd); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
I think you need this is because you never start the informers?  Probably worth a comment.